src/core/TransformControls.tsx

```diff
@@ -32,7 +33,15 @@ export type TransformControlsProps = ReactThreeFiber.Object3DNode<TransformContr
     onObjectChange?: (e?: THREE.Event) => void
   }
 
-export const TransformControls = React.forwardRef<TransformControlsImpl, TransformControlsProps>(
+export const TransformControls = React.forwardRef<TransformControlsImpl, TransformControlsProps>((props, ref) => {
+  const { mode } = props
+  if (mode === 'all') {
+    return <MultiTransformControls ref={ref as any} {...props} />
+  }
+  return <SingleTransformControls ref={ref as any} {...props} />
```

Comment:
FYI: To keep things clean I duplicated the original `TransformControls` into 2: `SingleTransformControls` and modified `MultiTransformControls`. After I have a better handle on how to implement all features in `MultiTransformControls` it may make sense to merge them into 1 component again, so long as the implementation is still readable and maintainable.